.buildkite/core.rayci.yml

```diff
@@ -71,7 +71,7 @@ steps:
         --except-tags debug_tests,asan_tests,post_wheel_build,ha_integration,mem_pressure,tmpfs,container,manual
 
   - label: ":ray: core: :windows: python tests"
-    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b" || build.env("RAYCI_CONTINUOUS_BUILD") == "1" || build.pull_request.labels includes "continuous-build"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b" || build.env("RAYCI_CONTINUOUS_BUILD") == "1"
```

Comment:
rather than using `build.env("BUILDKITE_PIPELINE_ID")`, just use `pipeline.id` ?

Reply:
see https://buildkite.com/docs/pipelines/conditionals#variable-and-syntax-reference-variables for supported builtin variables.

Reply:
Thank you! I've updated them all to use `pipeline.id`. Will merge the PR after the tests run.